packages/ajax/src/interceptors-cache.js

```diff
@@ -0,0 +1,361 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-param-reassign */
+
+import './typedef-cache.js';
+
+const SECOND = 1000;
+const MINUTE = SECOND * 60;
+const HOUR = MINUTE * 60;
+
+class LionCache {
+  constructor() {
+    this.expiration = +new Date() + HOUR;
+    /**
+     * @type {{[url: string]: CacheConfig }}
+     */
+    this.cacheConfig = {};
+
+    /**
+     * @type {{[url: string]: {date: number, data: object} }}
+     */
+    this.cacheObject = {};
```

Comment:
Isn't this a private property? In that case we should prefix it with `__`. Or perhaps even store it with a symbol, so that devs don't try to take things from the cache manually.

Using a `Map` is faster when it's accessed frequently.